integrationTest/providers.json

```diff
@@ -66,5 +66,13 @@
   "VULTR": {
     "token": "$VULTR_TOKEN",
     "domain": "$VULTR_DOMAIN"
+  },
+  "OVH": {
+    "COMMENT": "47-48 OVH doesn't support TLSA records with matching type 0 (full certificate)",
```

Comment:
Is there a matchingtype other than 0 that the tests could use so that the tests can run? (I don't know much about TLSA)

Reply:
Yes, matching type 2 is for sha512, but since they check the length of the checksum we'd have to have a string of 128 hex characters.
Do you want me to modify the integration test to check changing matching type to 2 instead of 0 ?

Reply:
Sure!  The extra 128 bytes won't fill my hard disk :-) . (but seriously... I'd rather have working tests than not)